profiler/profiler.go

```diff
@@ -130,6 +131,10 @@ type Config struct {
 	// defaults to false.
 	DebugLogging bool
 
+	// DebugLoggingOut is where the logger will write debug logs to, if enabled.
+	// It defaults to os.Stderr.
+	DebugLoggingOut io.Writer
```

Comment:
nit: Out seem short for Output which would mirror what is in stdlib: https://pkg.go.dev/log#SetOutput
Maybe be more explicit and use the full word

Reply:
Cool, sounds good to me! Done.